src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/util/Codes.kt

```diff
@@ -0,0 +1,4 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util
```

Comment:
could we move this to `uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies`
and update the usages inside that package to use these too?